src/super_gradients/training/datasets/samplers/all_samplers.py

```diff
@@ -2,10 +2,16 @@
 from super_gradients.training.datasets.samplers.infinite_sampler import InfiniteSampler
 from super_gradients.training.datasets.samplers.repeated_augmentation_sampler import RepeatAugSampler
 from torch.utils.data.distributed import DistributedSampler
-
+from torch.utils.data.sampler import Sampler, SequentialSampler, SubsetRandomSampler, RandomSampler, WeightedRandomSampler, BatchSampler
 
 SAMPLERS = {
     Samplers.INFINITE: InfiniteSampler,
     Samplers.REPEAT_AUG: RepeatAugSampler,
-    Samplers.DISTRIBUTED: DistributedSampler
+    Samplers.DISTRIBUTED: DistributedSampler,
+    Samplers.SAMPLER: Sampler,
```

Comment:
This is a base class and will crash the code if trying to use it.

Reply:
removed